include/RAJA/util/Operators.hpp

```diff
@@ -427,7 +427,7 @@ struct minimum : public detail::binary_function<Arg1, Arg2, Ret>,
   RAJA_HOST_DEVICE constexpr Ret operator()(const Arg1& lhs,
                                             const Arg2& rhs) const
   {
-    return (lhs < rhs) ? lhs : rhs;
+    return (rhs < lhs) ? rhs : lhs;
```

Comment:
Do you think this change should have a comment with it?

Reply:
Sure, I can add that in.